packages/legend-query-builder/src/stores/QueryLoaderState.ts

```diff
@@ -246,7 +246,7 @@ export class QueryLoaderState {
         name,
       )) as Query;
       this.onQueryRenamed?.(query);
-      this.applicationStore.notificationService.notify(
+      this.applicationStore.notificationService.notifySuccess(
```

Comment:
TODO: just making a comment here, probably would be great if we can bring the same duplicated name warning here

Reply:
Makes sense, added 👍